gordo/reporters/base.py

```diff
@@ -0,0 +1,61 @@
+import abc
+import pydoc
+
+from typing import Dict, Any
+
+from gordo.machine import Machine
+
+
+class BaseReporter(abc.ABC):
+    @abc.abstractmethod
+    def report(self, machine: Machine):
+        """Report/log the machine"""
+
+    def to_dict(self) -> dict:
```

Comment:
Can we not use the serializer here? (and with the load?)

Reply:
Lovely.